plugins/UsersManager/vue/src/UserEditForm/UserEditForm.vue

```diff
@@ -130,30 +138,29 @@
           </div>
           <div>
             <div class="form-group row" style="position: relative">
-              <div class="col s12 m6">
+              <div class="col s12 m6 save-button">
                 <SaveButton
-                    v-if="currentUserRole === 'superuser' || isAdd"
-                    :value="saveButtonLabel"
-                    :disabled="isAdd && (!firstSiteAccess || !firstSiteAccess.id)"
-                    :saving="isSavingUserInfo"
-                    @confirm="saveUserInfo()"
+                  v-if="currentUserRole === 'superuser' || isAdd"
+                  :value="saveButtonLabel"
+                  :disabled="isAdd && (!firstSiteAccess || !firstSiteAccess.id)"
+                  :saving="isSavingUserInfo"
+                  @confirm="inviteUser"
                 />
               </div>
-              <div class="col s12 m6">
-                <div v-if="isAdd" class="form-help">
-                     <span class="inline-help"
-                      v-html="$sanitize(
-                          translate('UsersManager_InviteSuccessNotification', [7]))"></span>
-                </div>
-              </div>
             </div>
-            <PasswordConfirmation
-              v-model="showPasswordConfirmationForInviteUser"
-              @confirmed="inviteUser"
+            <p class="resend-notes" v-if="user && isPending"
             >
-              <h2 v-html="$sanitize(inviteUserTitle)"></h2>
-              <p>{{ translate('UsersManager_ConfirmWithPassword') }}</p>
-            </PasswordConfirmation>
+              {{ translate('UsersManager_InvitationSent') }}
+              <span class="resend-link" @click="resendRequestedUser"
+                    v-html="$sanitize(translate('UsersManager_ResendInvite'))"></span>
+            </p>
+<!--            <PasswordConfirmation-->
+<!--              v-model="showPasswordConfirmationForInviteUser"-->
+<!--              @confirmed="inviteUser"-->
+<!--            >-->
+<!--              <h2 v-html="$sanitize(inviteUserTitle)"></h2>-->
+<!--              <p>{{ translate('UsersManager_ConfirmWithPassword') }}</p>-->
+<!--            </PasswordConfirmation>-->
```

Comment:
Please ALWAYS avoid commenting out code that shouldn't be used anymore. It does not have any benefit keeping such stuff. If it needs to be reverted at some point, that can be easily done using git history. No need for polluting code.